File: python codes/code_2.py
def team2():
    lst = [1, 2, 3, 4]
    for i in range(5):  # Logical Error: Out-of-range loop
        print(lst[i])  # Will cause IndexError
    print("You qualify for the next round!")  # Won't execute due to the error

team2()

File: cpp codes/code2.cpp
#include <iostream>
#include <vector>
using namespace std;

void team2() {
    vector<int> lst = {1, 2, 3, 4};
    for (int i = 0; i <= 4; i++) { // Logical Error: Out-of-range loop
        cout << lst.at(i) << endl; // Will cause runtime error
    }
    cout << "You qualify for the next round!" << endl;
}

int main() {
    team2();
    return 0;
}
